mix.exs

```diff
@@ -1,13 +1,27 @@
 defmodule TypoKiller.MixProject do
   use Mix.Project
 
+  @version "0.1.0"
+  @repo "https://github.com/samuelpordeus/typo_killer"
+
   def project do
     [
       app: :typo_killer,
-      version: "0.1.0",
-      elixir: "~> 1.9",
+      version: @version,
+      elixir: "~> 1.8",
```

Comment:
hey! why exactly we are changing elixir version to `1.8`? `dialyxir`? 🙂

Reply:
Oops, i just forgot to update, because my environment is for Elixir 1.8 >_<
I'll change

Reply:
Maybe i can add ASDF `.tool-versions` file to use Elixir `1.9`

Reply:
@aleDsz sounds like a good plan